go/tasks/plugins/array/k8s/config.go

```diff
@@ -32,13 +32,19 @@ var (
 	configSection = config.MustRegisterSection(configSectionKey, defaultConfig)
 )
 
+type NodeSelectorConfig struct {
+	key   string `json:"key", pflag:",Node Selector key.`
```

Comment:
I think Pod's NodeSelector supports multiple keys here right? and it does an AND between them or something?
we should match that behavior then

Reply:
agreed, like we do for interruptible: https://github.com/lyft/flyteplugins/blob/master/go/tasks/pluginmachinery/flytek8s/config/config.go#L51

Reply:
Thanks for the reference! Updated.